compliance/http/src/test/java/org/eclipse/rdf4j/http/server/ProtocolTest.java

```diff
@@ -197,6 +203,31 @@ public void testUpdateDirect_POST()
 	}
 
 	/**
+	 * Checks that the server accepts a formencoded POST with an update and a timeout parameter.
+	 */
+	@Test
+	public void testUpdateForm_POST()
+		throws Exception
+	{
+		String update = "delete where { <monkey:pod> ?p ?o . }";
+		String location = Protocol.getStatementsLocation(TestServer.REPOSITORY_URL);
+		CloseableHttpClient httpclient = HttpClients.createDefault();
+		HttpPost post = new HttpPost(location);
+		List<NameValuePair> nvps = new ArrayList<NameValuePair>();
+		nvps.add(new BasicNameValuePair(Protocol.UPDATE_PARAM_NAME, update));
+		nvps.add(new BasicNameValuePair(Protocol.TIMEOUT_PARAM_NAME, "1"));
+		UrlEncodedFormEntity entity = new UrlEncodedFormEntity(nvps, Charsets.UTF_8);
+
+		post.setEntity(entity);
+
+		CloseableHttpResponse response = httpclient.execute(post);
+
+		System.out.println("Update Form Post Status: " + response.getStatusLine());
+		int statusCode = response.getStatusLine().getStatusCode();
+		assertEquals(true, statusCode >= 200 && statusCode < 400);
+	}
+
+	/**
```

Comment:
Drat. spring-test (4.2.1) requires servlet-api 3.0, while we're still relying on 2.5. We should probably upgrade, but that means that we need to drop support for tomcat 6. I have no problem with that but I think we should not just do this as a quick fix in an unrelated issue, just to get our test rig working :) I'll raise a new issue.